rust-mode-tests.el

```diff
@@ -1377,24 +1372,6 @@ this_is_not_a_string();)"
      "r\" this is a comment\n" font-lock-comment-face
      "\"this is a string\"" font-lock-string-face)))
 
-(ert-deftest font-lock-raw-string-constant ()
-  ;; There was an issue in which a multi-line raw string would be fontified
-  ;; correctly if inserted, but then incorrectly if one of the lines was then
-  ;; edited.  This test replicates how font-lock responds when text in the
-  ;; buffer is modified in order to reproduce it.
```

Comment:
Oops, I was only talking about the `font-lock-raw-string-constant` test there.

The other test, `font-lock-extend-region-in-string`, explicitly called `rust-font-lock-extend-region`; which is gone, making the test irrelevant, I think.